components/modal/__tests__/modal.browser-test.jsx

```diff
@@ -106,11 +106,6 @@ describe('SLDSModal: ', function () {
 			modal = getModalNode(document.body);
 		});
 
-		it('renders a noscript', () => {
-			const renderedNoScriptNode = ReactDOM.findDOMNode(cmp);
-			expect(renderedNoScriptNode.firstChild.tagName).to.equal('NOSCRIPT');
```

Comment:
Latest version doesn't render a noscript tag anymore, so I'm removing this test. We could keep it and check for undefined, but didn't feel it was necessary.